src/openforms/forms/migrations/0076_move_confirmation_email_setting.py

```diff
@@ -0,0 +1,55 @@
+# Generated by Django 3.2.18 on 2023-04-14 08:38
+
+from django.db import migrations
+
+from openforms.forms.constants import ConfirmationEmailOptions
+
+
+def forward(apps, schema_editor):
+    """Change the setting for the confirmation email from a choices field to a boolean field"""
+    Form = apps.get_model("forms", "Form")
+
+    forms = Form.objects.all()
+
+    for form in forms:
+        form.send_confirmation_email = (
+            form.confirmation_email_option != ConfirmationEmailOptions.no_email
+        )
+
+    Form.objects.bulk_update(forms, fields=["send_confirmation_email"])
```

Comment:
```py
from django.db.models import Case, When, Value
Form.objects.update(
    send_confirmation_email=Case(
        When(confirmation_email_option=Value(ConfirmationEmailOptions.no_email), then=False), 
        default=True
    )
)
```
single query to do the update at the database level rather than python, see https://docs.djangoproject.com/en/3.2/ref/models/conditional-expressions/#advanced-queries

Reply:
Oooh I had tried to do it in one query with the `F()` but had not succeeded! Nice